apps/bajour/pages/mitmachen.tsx

```diff
@@ -29,8 +28,6 @@ export default function Mitmachen() {
           name: lastName as string | undefined,
           memberPlanSlug: memberPlanBySlug as string | undefined
         }}
-        successURL={`${locationOrigin}/profile/subscription`}
```

Comment:
I added a migration guide for the deployers: https://we-publish.gitbook.io/we.publish/developers/breaking-changes#id-2024-11-27-https-github.com-wepublish-wepublish-pull-1655